mavenplugin/pom.xml

```diff
@@ -29,6 +29,7 @@
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
             <version>3.3.9</version>
+            <scope>provided</scope>
```

Comment:
What's the reasoning for updating the scope here?

Reply:
I made the change to fix a warning message at compile time, but I reverted it.